test/stats.bats

```diff
@@ -39,3 +39,48 @@ function teardown() {
     echo $JSON | jq -e '.stats[0].memory.workingSetBytes.value > 0'
     [ "$status" -eq 0 ]
 }
+
+@test "container stats" {
+    # given
+    container2config=$(cat "$TESTDATA"/container_redis.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["name"] = ["podsandbox1-redis2"];obj["metadata"]["name"] = "podsandbox1-redis2"; json.dump(obj, sys.stdout)')
+    echo "$container2config" > "$TESTDIR"/container_redis2.json
+    run crictl runp "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr1_id="$output"
+    run crictl create "$pod_id" "$TESTDIR"/container_redis2.json "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr2_id="$output"
+    run crictl start "$ctr1_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run crictl start "$ctr2_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    # when
+    run crictl stats -o json "$ctr1_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr1_stats_JSON="$output"
+
+    run crictl stats -o json "$crt2_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr2_stats_JSON="$output"
+
+    echo $ctr1_stats_JSON | jq -e '.stats[0].memory.workingSetBytes.value'
+    [ "$status" -eq 0 ]
+    ctr1_memory_bytes="$output"
```

Comment:
Similar thing here -- `$output` is only set by bats' `run`.

Reply:
fixed!